draft-ietf-jsonpath-base.md

```diff
@@ -289,50 +298,52 @@ JSONPath expressions are applied to a JSON value, the *argument*.
 Within the JSONPath expression, the abstract name `$` is used to refer
 to the *root node* of the argument, i.e., to the argument as a whole.
 
-JSONPath expressions can use the *dot notation*
+JSONPath expressions use the *bracket notation*, for example:
 
 ~~~~
-$.store.book[0].title
+$['store']['book'][0]['title']
 ~~~~
 
-or the more general *bracket notation*
+or the more compact *dot notation*, for example:
 
 ~~~~
-$['store']['book'][0]['title']
+$.store.book[0].title
 ~~~~
 
 to build paths that are input to a JSONPath implementation.
+A single path may use a combination of bracket and dot notations.
+
+A wildcard `*` ({{wildcard}}) may be used to select all children of an
```

Comment:
wildcards are also available for all descendants.

This should be fixed in the context of making wildcards selectors.

Reply:
I believe that context is the current PR. The wildcard selector is defined in section 3.5.2.

I'll have to think about how best to address this because a selector acts on a single node and selects zero or more of its children. A selector _per se_ never selects other descendants. But at this stage in the document, we aren't ready to talk about segments and selectors, so we may need to fudge the wording here.

Reply:
See the rewording in e6ef8e8460b2c5a6f629811f8d8728f790bf5d86.

Reply:
OK, that works, but could be improved later.  Keeping this open.